var/.htaccess

```diff
@@ -0,0 +1,2 @@
+Order deny,allow
+Deny from all
```

Comment:
It's only Apache2.2 compatible. Watch out with Apache2.4: `Require all denied`

Reply:
this need to be updated everywhere or nowhere :)